crypto/util.go

```diff
@@ -0,0 +1,41 @@
+package crypto
+
+import (
+	"encoding/asn1"
+	"fmt"
+	"math/big"
+
+)
+
+// parseSignature parses an asn1 stucture (R,S) into a slice of bytes as required by the `Siger.Sign` method.
+func ParseSignature(kmsSignature []byte, curve SignatureAlgorithm) ([]byte, error) {
```

Comment:
Should this be part of kms package instead? is it used anywhere else?

Reply:
added a related comment here https://github.com/onflow/flow-go-sdk/pull/340#discussion_r1101941778